langchain.gemspec

```diff
@@ -29,10 +29,10 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "pry-byebug", "~> 3.10.0"
+  spec.add_development_dependency "cohere-ruby", "~> 0.9.3"
```

Comment:
Do you think we should move ALL of the gems here or maybe leave some **core** ones under actual deps below?

Reply:
The idea is that you can swap out different LLMs. Ideally, eventually, I think we ought to have a lot more LLM options to pick from.